### PR TITLE
Update readme, splashes, and credits. fix gold armor. change chainmail to crystal guard armor. warped door item, wandering trader added.

### PR DESCRIPTION
**Pull request by VelvetRemedy**
wandering trader and warped door made by bronydog

### CODE REVIEW

**Comment by Sollace** [comment]
Should we think about not modifying the credits.txt, in the event Mojang changes them again, and use the readme.md from the repo instead?

File: assets/minecraft/lang/eq_eq-full.json [hunk 1]
```diff
@@ -77,10 +77,10 @@
   "item.minecraft.leather_chestplate": "CMC Cape",
   "item.minecraft.leather_leggings": "CMC Knee pads",
   "item.minecraft.leather_boots": "CMC Boots",
-  "item.minecraft.chainmail_helmet": "Gummy Helmet",
```
Comment (Sollace) [inline assets/minecraft/lang/eq_eq-full.json:80]: Lol, gummy. Could we maybe turn the turtle helmet into a gummy helmet?

Comment (PCelestia) [reply assets/minecraft/lang/eq_eq-full.json:80]: hmm maybe lol

**Comment by PCelestia** [comment]
credits shouldn't change often, so i think its ok for now. though i agree we should probably find some other way to do it

**Review by PCelestia — APPROVED** [review]

**Pull request merged**